server/functions/helpers.lua

```diff
@@ -4,14 +4,13 @@
 ---@param group string - Player group required to use the command
 ---@param cb function - Callback function
 ---@param suggestions table
----@param rcon boolean
-ATL.RegisterCommand = function(name, description, group, cb, suggestions, rcon)
```

Comment:
Not bad but I assume that you wrote somewhere that this might be included again.